src/main/webapp/components/directives/cache/replicated-cache.html

```diff
@@ -0,0 +1,47 @@
+<div>
+  <tabset>
+    <tab heading="General" active="true">
+      <generalreplicated data='currentCache' metadata="resolveDescription('general')" init-defaults="initDefaults" read-only="readOnly"/>
+    </tab>
+    <tab heading="Locking" active="false">
```

Comment:
Just an idea: you might consider using ng-repeat here, where you would give ng-repeat list of objects with information for each tab! It may or may not be worth trouble to prepare this objects, I am not sure about what data is available and how is it formatted, but it would reduce some html repetition (and therefore also chances of mistake).
Same idea goes for tabs below!